src/pages/payment-methods/_mobile-accordian-item.tsx

```diff
@@ -51,7 +51,9 @@ const MobileAccordianItem = ({ locale, pd }: MobileAccordianItemProps) => {
             {pd.note && (
                 <Notes>
                     <Header type="paragraph-1" weight="normal">
-                        {localize('Note:')} {pd.note}
+                        <Localize translate_text="_t_Note:_t_" />
+                        <span>&nbsp;</span>
```

Comment:
We exactly need this?

Reply:
Yes, to give a space between colon (:) and note

Reply:
Instead of creating new element to the dom can we use this? `{' '}`